packages/contracts/test/core/permission/permission-manager.ts

```diff
@@ -579,15 +579,9 @@ describe('Core: PermissionManager', function () {
         },
       ];
 
-      await expect(pm.bulkOnMultiTarget(bulkItems)).to.be.revertedWith(
-        customError(
-          'Unauthorized',
-          pm.address,
-          signers[1].address,
-          signers[0].address,
-          ADMIN_PERMISSION_ID
-        )
-      );
+      await expect(pm.connect(signers[2]).bulkOnMultiTarget(bulkItems)).to.be.revertedWith(
```

Comment:
The test is still skipped. Please remove the `.skip` and try if the test runs now.

Also, please remove the `//TODO` unless something is still wrong with this test.

Please reformulate the test description. 
> should revert on errors

is not clear enough. I would write 

> throws the `Unauthorized` error when the caller does not have the `ROOT_PERMISSION_ID` permission

It would also be good to format the file with prettier as the default formatter.